merlin-server/sql/merlin/tables/constraint_run.sql

```diff
@@ -0,0 +1,49 @@
+create type constraint_status as enum('resolved', 'constraint-outdated', 'simulation-outdated');
```

Comment:
I believe you mentioned extracting this statement out to a file in `sql/merlin/domain-types`, alongside some of the other Merlin "run status" enums.